recipes/packit/version.patch

```diff
@@ -0,0 +1,8 @@
+--- setup.py	2019-06-17 17:23:14.000000000 -0400
++++ setup.py.new	2019-07-31 14:36:01.000000000 -0400
+@@ -1,5 +1,5 @@
+ from setuptools import setup
+
+
+-setup(setup_requires=['pbr>=4.0', 'wheel>=0.24', 'glob2==0.4.1', 'packaging==16.8'], pbr=True, long_description_content_type="text/markdown")
++setup(setup_requires=['pbr>=4.0', 'wheel>=0.24', 'glob2>=0.4', 'packaging>=16.8'], pbr=True, long_description_content_type="text/markdown")
```

Comment:
I think you should raise an error on this project about it the ``glob2`` and ``packaging``

Reply:
And ``glob2>=0.4.1`` should be like that right?

Reply:
Yes, you're right